cosmetics-web/app/helpers/component_build_helper.rb

```diff
@@ -1,7 +1,7 @@
 module ComponentBuildHelper
   def cmr_errors(component)
-    component.cmrs.each_with_index.flat_map { |cmr, index| cmr.errors.map { |attr, msg| { text: msg, href: "#component_cmrs_attributes_#{index}_#{attr}" } } } +
-      component.errors.reject { |attr, _| attr.to_s.include? "cmrs." }.map { |_, msg| { text: msg, href: "#component_cmrs_attributes_0_name" } }
+    component.cmrs.each_with_index.flat_map { |cmr, index| cmr.errors.map { |error| { text: error.message, href: "#component_cmrs_attributes_#{index}_#{error.attribute}" } } } +
+      component.errors.reject { |_error| attribute.to_s.include? "cmrs." }.map { |error| { text: error.message, href: "#component_cmrs_attributes_0_name" } }
```

Comment:
:raising_hand_man:  
Not 100% sure on this but better safe than sorry. From what I see in the changes now the interface is `error#attribute`  (as used in line 3). I belive the line 4 could be wrong then:
```suggestion
      component.errors.reject { |error| error.attribute.to_s.include? "cmrs." }.map { |error| { text: error.message, href: "#component_cmrs_attributes_0_name" } }
```

Reply:
Good catch!